.github/workflows/client_node.yaml

```diff
@@ -0,0 +1,63 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Run Node Client Generator and Create Pull Request
+on:
+  push:
+    branches:
+      - "master"
+jobs:
+  create_pr:
+    runs-on: ubuntu-18.04
+    steps:
+    - name: Checkout this repo
```

Comment:
Do you need to checkout this repo?

Reply:
Yes, this repo contains schema files used for generation.